src/main/drivers/pwm_output.c

```diff
@@ -134,29 +134,9 @@ static void pwmWriteUnused(uint8_t index, float value)
     UNUSED(value);
 }
 
-static void pwmWriteBrushed(uint8_t index, float value)
-{
-    *motors[index].ccr = lrintf((value - 1000) * motors[index].period / 1000);
-}
-
 static void pwmWriteStandard(uint8_t index, float value)
 {
-    *motors[index].ccr = lrintf(value);
-}
-
-static void pwmWriteOneShot125(uint8_t index, float value)
-{
-    *motors[index].ccr = lrintf(value * ONESHOT125_TIMER_MHZ/8.0f);
-}
-
-static void pwmWriteOneShot42(uint8_t index, float value)
-{
-    *motors[index].ccr = lrintf(value * ONESHOT42_TIMER_MHZ/24.0f);
-}
-
-static void pwmWriteMultiShot(uint8_t index, float value)
-{
-    *motors[index].ccr = lrintf(((value-1000) * MULTISHOT_20US_MULT) + MULTISHOT_5US_PW);
+    *motors[index].ccr = lrintf((((value - 1000) / 1000) * motors[index].pulseLen) + motors[index].pulseMin);
```

Comment:
This is quite time-critical code - `motors[index].pulseLen` should factor in the ` / 1000` part.
And - not that important - pulseMin may contain ` - 1000 * pulseLen` : 
`lrintf(value  * motors[index].pulseScale + motors[index].pulseOffset)`